<launch>
<node name="packing_and_unpacking_node" pkg="dtvr_message_packing_unpacking" type="dtvr_message_packing_unpacking_node" output="screen">
                                         <!--packing-->
      <param name="SourceID" value="1"/>
      <param name="TargetID" value="201"/>
      <param name="DtObjectID" value="201"/>
      <param name="CloudMessagePubTopic" value="/R_UAV_0/Message_to_Cloud"/>
      <param name="GlobalPositionMessageSubTopic" value="/mavros/global_position/global"/>
      <param name="LocalPositionMessageSubTopic" value="/mavros/local_position/pose"/>
      <param name="LocalVelocityMessageSubTopic" value="/mavros/local_position/velocity"/>
      <param name="UavStateMessageSubTopic" value="/mavros/state"/>
      <param name="BatteryMessageSubTopic" value="/mavros/battery"/>
      <param name="MessagePubFrequency" value="10"/>

                                         <!--unpacking-->
      <param name="CloudMessageSubTopic" value="/R_UAV_0/Message_from_Cloud"/>
      <param name="TargetVelocityMessagePubTopic" value="/R_UAV_0/Target/Velocity"/>
      <param name="TargetPositionMessagePubTopic" value="/R_UAV_0/Target/Position"/>
      <param name="ArmCommandMessagePubTopic" value="/R_UAV_0/Set/Arm"/>
      <param name="TargetFModeMessagePubTopic" value="/R_UAV_0/Set/FMode"/>
      <param name="VRControlMessagePubTopic" value="/R_UAV_0/VR/Control"/>
      <param name="ComputerCmdMessagePubTopic" value="/R_UAV_0/Computer/Cmd"/>
      <param name="ApplyCameraMessagePubTopic" value="/R_UAV_0/Apply/Camera"/>
</node>
</launch>
